app/controllers/puppetca_controller.rb

```diff
@@ -4,7 +4,7 @@ def index
     # expire cache if forced
     Rails.cache.delete("ca_#{@proxy.id}") if params[:expire_cache] == "true"
     certs = find_certs
-    @certificates = certs.sort.paginate :page => params[:page], :per_page => Setting::General.entries_per_page
+    @certificates = certs.sort
```

Comment:
yes i understand, imho you should set the default sort on the js side now, as it has no knowledge on which field sorting was done